apps/docs/components/collapsible/Collapsible.tsx

```diff
@@ -1,54 +1,31 @@
 "use client";
 
+import { CollapseIcon, Icon } from "@/components/icon";
 import clsx from "clsx";
-import { type ReactNode, useEffect, useRef, useState } from "react";
-import { ToggleButton } from "react-aria-components";
-import { Icon, CollapseIcon } from "@/components/icon";
+import type { ReactNode } from "react";
+import { Button, UNSTABLE_Disclosure as Disclosure, UNSTABLE_DisclosurePanel as DisclosurePanel, type DisclosureProps } from "react-aria-components";
 
 import "./collapsible.css";
 
-export interface CollapsibleProps {
+export interface CollapsibleProps extends DisclosureProps {
     children: ReactNode;
```

Comment:
you should be able to remove children and className from here since it's defined inside DisclosureProps and DisclosureProps allows the use of renderProps

Reply:
I see you remove className but not children?